CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## [Unreleased](https://github.com/idealista/yatm/tree/develop)
+### Fixed
```

Comment:
As this PR is a hotfix, shouldn't the changelog be updated to a fixed version (2.1.1)?